.github/workflows/test.yml

```diff
@@ -0,0 +1,77 @@
+name: Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'develop'
+      - 'main'
+      - 'release/**'
+      - 'feature/ci'
+  pull_request:
+    branches:
+      - 'develop'
+      - 'main'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        node-version: ['20.x']
```

Comment:
If we're only testing one version of node, it doesn't need to be in a matrix.

Reply:
Makes sense :)